snapcraft/cli/env.py

```diff
@@ -0,0 +1,20 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+
+
+def is_containerbuild():
+    return os.environ.get('SNAPCRAFT_CONTAINER_BUILDS')
```

Comment:
Thinking out loud... are you planning on adding other variables here? I'm wondering about namespacing since this one is about LXD - on the other hand it's a general flag that's handled before the actual LXD code comes into play.